### PR TITLE
Update to Hanami 2.1.0.beta2, use ERB for views

### PR DESCRIPTION
**Pull request by timriley**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,13 +24,16 @@ gem "sequel"
 
 group :cli, :development, :test do
   # Hanami testing helpers
-  gem "hanami-rspec"
+  gem "hanami-rspec", github: "hanami/rspec", branch: "main"
```
Comment (katafrakt) [inline Gemfile:27]: just curious - why is this needed?

Comment (timriley) [reply Gemfile:27]: @katafrakt It wasn't strictly necessary; there haven't been updates to that gem for the 2.1 cycle. But I was just taking the approach of making sure _everything_ was running on main, for consistency.
